<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>565</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="settings_gb">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTreeWidget" name="model_tree">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <attribute name="headerMinimumSectionSize">
         <number>50</number>
        </attribute>
        <column>
         <property name="text">
          <string>Property</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Value</string>
         </property>
        </column>
        <item>
         <property name="text">
          <string>Color</string>
         </property>
         <item>
          <property name="text">
           <string>Unit Color</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Min Color</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Max Color</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>Grid</string>
         </property>
         <item>
          <property name="text">
           <string>X Grid Number</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Z Grid Number</string>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>Distance</string>
         </property>
         <item>
          <property name="text">
           <string>X Min</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>X Max</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Z Min</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Z Max</string>
          </property>
         </item>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="save_gb">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string>Output</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="1" column="0">
          <widget class="QLineEdit" name="lineSource"/>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="lineStat"/>
         </item>
         <item row="5" column="0">
          <widget class="QLineEdit" name="lineVel2d"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Source (*.src)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Station (*.stat)</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>2D Velocity (*.vel2d)</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="buttonSave">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="buttonVel">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="buttonStat">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="buttonSource">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="save_expandbutton">
        <property name="font">
         <font>
          <family>Calibri</family>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QToolButton { border: none; }</string>
        </property>
        <property name="text">
         <string>Save Model</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="arrowType">
         <enum>Qt::RightArrow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="view_gb">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
      </font>
     </property>
     <property name="title">
      <string>View</string>
     </property>
     <layout class="QVBoxLayout" name="layoutViewGroupBox">
      <item>
       <layout class="QHBoxLayout" name="buttonLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonPickStation">
          <property name="text">
           <string>set stations</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonPickSource">
          <property name="text">
           <string>set events</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
